src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -136,60 +129,18 @@ public void resolveRequest(@NotNull final IRequestManager manager, @NotNull fina
                     }
                 }
             }
-
-            final List<PlayerEntity> players = new ArrayList<>(colony.getMessagePlayerEntitys());
-            final PlayerEntity owner = ServerUtils.getPlayerFromUUID(colony.getWorld(), ((Colony) colony).getPermissions().getOwner());
-            final StringTextComponent colonyDescription = new StringTextComponent(colony.getName() + ":");
-
-            final ILocation requester = request.getRequester().getLocation();
-            final IBuilding building = colony.getBuildingManager().getBuilding(requester.getInDimensionLocation());
-
-            if (building == null || (building.getCitizenForRequest(request.getId()).isPresent() && !building.getCitizenForRequest(request.getId())
-                                                                                                      .get()
-                                                                                                      .isRequestAsync(request.getId())))
-            {
-                if (manager.getColony().getWorld().isDaytime())
-                {
-                    if (owner != null)
-                    {
-                        players.remove(owner);
-
-                        LanguageHandler.sendPlayerMessage(owner, "com.minecolonies.requestsystem.playerresolver",
-                          request.getRequester().getRequesterDisplayName(manager, request).getFormattedText(),
-                          getRequestMessage(request).getFormattedText(),
-                          request.getRequester().getLocation().toString()
-                        );
-                    }
-                    LanguageHandler.sendPlayersMessage(players, "com.minecolonies.requestsystem.playerresolver",
-                      colonyDescription.getFormattedText() + " " + request.getRequester().getRequesterDisplayName(manager, request).getFormattedText(),
-                      getRequestMessage(request).getFormattedText(),
-                      request.getRequester().getLocation().toString());
-                }
-            }
-
         }
         assignedRequests.add(request.getId());
     }
 
-    private ITextComponent getRequestMessage(@NotNull final IRequest request)
```

Comment:
Why was this removed?

Reply:
because we used it to notify the player before, since we don't do this anymore, we don't need this anymore.